src/Proxy/AbstractProxy.php

```diff
@@ -204,11 +205,13 @@ protected function prepareArgsLine(ReflectionFunctionAbstract $functionLike)
      */
     protected function getOverriddenFunction(ReflectionFunctionAbstract $functionLike, $body)
     {
-        $reflectionReturnType = PHP_VERSION_ID >= 50700 ? $functionLike->getReturnType() : '';
+        $reflectionReturnType = PHP_VERSION_ID >= 70000 ? $functionLike->getReturnType() : '';
         $modifiersLine        = '';
         if ($reflectionReturnType) {
-            $nsPrefix             = $reflectionReturnType->isBuiltin() ? '' : '\\';
-            $reflectionReturnType = $nsPrefix . (string)$reflectionReturnType;
+            $nullablePrefix = $reflectionReturnType->allowsNull() ? '?' : '';
+            $nsPrefix       = $reflectionReturnType->isBuiltin() ? '' : '\\';
+
+            $reflectionReturnType = $nullablePrefix . $nsPrefix . (string) $reflectionReturnType;
```

Comment:
@lisachenko Are you sure that this works as desired? I havn't tested the code but in case I am using PHP 7.0.x, have a method with a return type and a signature like "($parameter = null)", a nullable operator will be added. This would break the code because nullable were added in PHP 7.1 like mentioned in the title. Missed I something?